### DISCUSSION TITLE
Can not set `alias` property on request intercept

### DISCUSSION THREAD

**Discussion by WilsonCazarre** [cypress-io/cypress#17751] [Questions and Help]
I'm using Cypress to intercept requests made to a graphql api.
But for some reason, the alias that I'm setting is just not there.
```ts
export const hasOperationName = (
  req: { alias?: string; body?: any },
  operationName: string
) => {
  const { body } = req;
  return (
    body.hasOwnProperty("operationName") && body.operationName === operationName
  );
};

// Alias query if operationName matches
export const aliasQuery = (
  req: { alias?: any; body?: any },
  operationName: string
) => {
  if (hasOperationName(req, operationName)) {
    req.alias = `gql${operationName}Query`;
    console.log(req);
  }
};
```

This is basically the example code that is given in the [docs](https://docs.cypress.io/guides/testing-strategies/working-with-graphql#Alias-multiple-queries-or-mutations).

When I `console.log(req)` the object has no alias property and if it got into that `if` statement, it should have.
So basically, no alias is being set apparently:
![image](https://user-images.githubusercontent.com/46715026/129611032-5ce095b9-514f-490a-b0c3-c13e6ab267d8.png)
